heralding/heralding.yml

```diff
@@ -122,3 +122,4 @@ capabilities:
     timeout: 30
     protocol_specific_data:
       banner: "Microsoft ESMTP MAIL service ready"
+      fqdn: ""
```

Comment:
A comment should be added here, stating that if the fqdn options is commented out the fully qualified domain named of the host will be used...